helm_deploy/values-dev.yaml

```diff
@@ -46,6 +46,8 @@ generic-service:
    CURIOUS_URL: https://testservices.sequation.net/sequation-virtual-campus2-api/
    SYSTEM_PHASE: DEV
    BVL_URL: https://book-video-link-dev.prison.service.justice.gov.uk
+   BOOK_A_VIDEO_LINK_API_ENABLED: "true"
+   BOOK_A_VIDEO_LINK_API_URL: https://book-a-video-link-api-dev.prison.service.justice.gov.uk
    MOIC_URL: https://dev.moic.service.justice.gov.uk
```

Comment:
This means it will start to use our new service in DEV? Just wondering if this is going to confuse the guys and gals looking after the current BVL?

Reply:
Wonder if nest to leave it off for now?  We can show/demo locally.